Source/Fluids/WarpXFluidContainer.H

```diff
@@ -0,0 +1,172 @@
+/* Copyright 2023 Grant Johnson, Remi Lehe
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+#ifndef WARPX_WarpXFluidContainer_H_
+#define WARPX_WarpXFluidContainer_H_
+
+#include "Evolve/WarpXDtType.H"
```

Comment:
Do we actually need this include? (i.e. does the code compile without it?)

Reply:
We don't removed in the newest commit.